src/modules/updateStatus.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-undef */
+/* eslint-disable linebreak-style */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] As the previous reviewer said, kindly fix also this part. set up linters locally and run this command `npx eslint . --fix`